cmp/layout/Box.js

```diff
@@ -23,7 +25,6 @@ export class Box extends Component {
     render() {
         let {children, ...props} = this.getNonLayoutProps();
         props = merge(
-            {style: {display: 'flex', overflow: 'hidden', position: 'relative'}},
             {style: this.getLayoutProps()},
```

Comment:
wasn't this position: 'relative' important for something?

Reply:
I also would like to know what that's about in general - we could add a comment to its new home in the CSS.

I don't see why it should be something that an app would depend on being set for box if we are OK with its new role as "div with layout support," which I think remains a reasonable and easy-enough to explain reason for its existence.